plugins/texteditor.koplugin/main.lua

```diff
@@ -96,10 +99,10 @@ function TextEditor:getSubMenuItems()
     self.whenDoneFunc = nil -- discard reference to previous TouchMenu instance
     local sub_item_table = {
         {
-            text = _("Text editor settings"),
+            text = _("Settings"),
             sub_item_table = {
                 {
-                    text = _("Set text font size"),
+                    text = _("Text font size"),
```

Comment:
I'd make it either just:
```suggestion
                    text = _("Font size"),
```
Or:
```suggestion
                    text = _("Editor font size"),
```

Reply:
I made it unified with
https://github.com/koreader/koreader/blob/861214ce2ab3c820ba2475b7ea4207a7cfdde888/frontend/apps/filemanager/filemanagermenu.lua#L192
https://github.com/koreader/koreader/blob/e33476996acfa9aa2dab2c6e35c8eda4545100ce/frontend/apps/reader/modules/readertoc.lua#L1027
https://github.com/koreader/koreader/blob/e33476996acfa9aa2dab2c6e35c8eda4545100ce/frontend/apps/reader/modules/readerbookmark.lua#L99
In the text edtor there are no other items but text. Looks short and clear.

Reply:
> I'm fine with either "Text font size" or "Font size" - but not with "Editor font size" (which, for me, could evoke the editor title and buttons font size).

That's exactly my rationale, except "text" could be anything and "editor" specifically means the editor component.

Reply:
We have much worse cases in the font size dialogs e.g.
https://github.com/koreader/koreader/blob/bf6c0cdd6c5b22bbb38497b5df8abe428eb80307/frontend/apps/reader/modules/readercoptlistener.lua#L316

Reply:
Are you saying the size of that status bar and the font size aren't interlinked? Otherwise there's nothing that can be confusing about it.

Reply:
NB It might just as well be that `cre_header_status_font_size` is misleading and that the text is completely accurate. You'd have to check crengine code to see.

Reply:
The menu item `Font size` invokes the dialog titled `Size of top status bar`.
It's not good imho even if the font size and bar height are interlinked.

Reply:
Apologies, I misunderstood. I don't quite understand its relevance to the discussion here though. ^_^

Reply:
@poire-z said "editor font size" evokes the size of the title and buttons.

I said text could be anything.

And then I thought you said something like "this one here doesn't even say text!"